lib/mix/test/fixtures/umbrella_test/apps/bar/mix.exs

```diff
@@ -7,7 +7,8 @@ defmodule Bar.MixProject do
       version: "0.1.0",
       # Choose something besides *_test.exs so that these test files don't
       # get accidentally swept up into the actual Mix test suite.
-      test_pattern: "*_tests.exs"
+      test_pattern: "*_tests.exs",
+      test_coverage: [ignore_modules: [Bar.Protocol]]
```

Comment:
Can you ignore `Bar` instead of `Bar.Protocol`? Protocols has specific coverage features so we need to verify it. `Bar` is just a plain module and we already have another plain module as `Foo`. Thanks!